### PR TITLE
with-stitches example fixes #28650

### LINKED ISSUE #28650: Witth-stitches example won't compile
### What example does this report relate to?

with-stitches

### What version of Next.js are you using?

11.1.2

### What version of Node.js are you using?

16.7.0

### What browser are you using?

Firefox

### What operating system are you using?

macOS

### How are you deploying your application?

next start

### Describe the Bug

When trying spin up the with-stitches example I get this error message:

```TypeError: (0 , _stitches_react__WEBPACK_IMPORTED_MODULE_0__.createCss) is not a function```

I assume that this is due to changes in the Stitches API since the example was created.

### Expected Behavior

The example to work first time

### To Reproduce

Run `yarn create next-app --example with-stitches with-stitches-app` and spin the dev instanse as usual with `yarn dev`

### PR DESCRIPTION
**Pull request by RobEasthope**
The with-stitches example is broken presumably due to changes in the Stitches API now that it's hit `1.0.0`. I've created an issue already: #28650 

I've made three small changes:
- Replaced the createCss function in `stitcheds.config.js` with createStitches
- Replaced the getCssString function with getCssText  in `stitches.config.js` and `pages/_documents.jsx`
- Bumped `@stitches/react` to `^1.0.0` 🎉

I've left the rest of the example code alone as I'm still exploring the library after seeing yesterdays release annoucement.

@peduarte Is there anything else that needs to updated?

### CODE REVIEW

File: examples/with-stitches/stitches.config.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import { createCss } from '@stitches/react'
+import { createStitches } from '@stitches/react'
 
-export const { css, styled, global, getCssString } = createCss({
+export const { css, styled, globalCss, getCssText } = createStitches({
```
Comment (peduarte) [inline examples/with-stitches/stitches.config.js:3]: Can we also export `theme`, `createTheme`, and `config`? Thanks!

**Comment by peduarte** [comment]
Hey, thanks for this!

The utils API [has changed also](https://stitches.dev/blog/migrating-from-beta-to-v1#updated-utils-api). So that'll need to be updated in the config.

File: examples/with-stitches/stitches.config.js (hunk 1 above)
Comment (RobEasthope) [reply examples/with-stitches/stitches.config.js]: @peduarte It's done 👍 

Thanks for all the work you and your team have put in on Stitches and Radux UI. They make a great combo 😃

**Comment by RobEasthope** [comment]
> Hey, thanks for this!
> 
> The utils API [has changed also](https://stitches.dev/blog/migrating-from-beta-to-v1#updated-utils-api). So that'll need to be updated in the config.

No worries, I've updated the utils syntax as well

**Review by peduarte — APPROVED** [review]
Nice. LGTM! 🚀

**Review by leerob — APPROVED** [review]
Thank you!

**Pull request merged**